team_connectors_details.go

```diff
@@ -0,0 +1,81 @@
+package fivetran
+
+import (
+    "context"
+    "encoding/json"
+    "fmt"
+)
+
+// TeamConnectorsDetailsService implements the Team Management, Retrieve connector membership
+// Ref. https://fivetran.com/docs/rest-api/teams#retrieveconnectormembership
+type TeamConnectorsDetailsService struct {
```

Comment:
```suggestion
type TeamConnectorMembershipDetailsService struct {
```
and all relevant renamings.